appdev/guides/contenthub.rst

```diff
@@ -0,0 +1,157 @@
+=============================================
+The content hub. Tips and tricks
+=============================================
+.. figure:: /_static/images/appdev/guides/contenthubimages/01.png
+        :align: center
+
+        Content Hub Share Page
+
+On Ubuntu Touch the apps are confined. The way of sharing files between them is through the Content Hub, a part of the system that takes care of file import, export and sharing.
+
+Different ways of sharing the content
+-------------------------------------
+As we can see in the `documentation <https://api-docs.ubports.com/sdk/apps/qml/Ubuntu.Content/ContentHandler.html#detailed-description>`_, there are several ways of handling the file to be shared:
+
+- ContentHandler.Source (The app selected is going to be the source of the file imported)
+- ContentHandler.Destination (The app selected is going to be the destination of the file exported)
+- ContentHandler.Share (The app selected is going to be the app from witch the file is going to be shared from)
+
+Importing
+^^^^^^^^^
+.. figure:: /_static/images/appdev/guides/contenthubimages/02.png
+        :align: center
+
+        Webapp Creator on the OpenStore.io
```

Comment:
I think we normally call the store `OpenStore` rather than `OpenStore.io`.

Reply:
Oh, that was for the Medium page... silly me